From 0c7a4d8764d622303a54f177e4ae525d32939232 Mon Sep 17 00:00:00 2001
From: Bogdan-Gabriel Roman <bogdan-gabriel.roman@nxp.com>
Date: Fri, 28 Jan 2022 15:59:52 +0200
Subject: [PATCH 1/6] mk: add PLAT_${S32_PLAT_SOC} variable

This new variable will help set common SOC configurations for when
boards will be used as build targets instead of the now-used SOCs
targets.

Issue: ALB-8443
Upstream-Status: Pending 

Signed-off-by: Bogdan-Gabriel Roman <bogdan-gabriel.roman@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 plat/nxp/s32/s32_common.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/plat/nxp/s32/s32_common.mk b/plat/nxp/s32/s32_common.mk
index 2a83faac9..6e476c9d4 100644
--- a/plat/nxp/s32/s32_common.mk
+++ b/plat/nxp/s32/s32_common.mk
@@ -19,6 +19,10 @@ ERRATA_SPECULATIVE_AT	:= 1
 HEXDUMP ?= xxd
 SED ?= sed
 
+ifneq ($(S32_PLAT_SOC),)
+$(eval $(call add_define_val,PLAT_$(S32_PLAT_SOC)))
+endif
+
 S32G_EMU		?= 0
 $(eval $(call add_define_val,S32G_EMU,$(S32G_EMU)))
 
-- 
2.17.1

